Gradle Won't Build

Gradle commands all fail with mysterious errors that don't make sense and seem like it isn't able to find/see/import forge. There is no real issue with the OreGen class. I have verified that those imports and features exist in 1.15.2 and are being used correctly (they also worked flawlessly in 1.14.4). The issue seems to be with the Gradle and project configuration but I'm not sure what. 

With the current configuration, all gradle builds fail with an error something like:
```> Configure project :
Deprecated MinecraftExtension.setMappings called. Use mappings(channel, version)
New Dep: net.minecraftforge:forge:1.15.2-31.2.0_mapped_snapshot_20200519-1.15.1

> Task :compileJava FAILED
/home/noah/code/morefuelsmod/src/main/java/io/morefuelsmod/MFMOreGen.java:75: error: cannot find symbol
                                        biome.addFeature(GenerationStage.Decoration.UNDERGROUND_ORES, Biome.createDecoratedFeature(Feature.ORE, new OreFeatureConfig(stoneType, wantedBlock.getDefaultState(), veinSize), Placement.COUNT_RANGE, cfg));
                                                                                                           ^
  symbol:   method createDecoratedFeature(Feature<OreFeatureConfig>,OreFeatureConfig,Placement<CountRangeConfig>,CountRangeConfig)
  location: class Biome
/home/noah/code/morefuelsmod/src/main/java/io/morefuelsmod/MFMOreGen.java:96: error: cannot find symbol
                        biome.addFeature(GenerationStage.Decoration.UNDERGROUND_ORES, Biome.createDecoratedFeature(Feature.ORE, new OreFeatureConfig(stoneType, wantedBlock.getDefaultState(), veinSize), Placement.COUNT_RANGE, cfg));
                                                                                           ^
  symbol:   method createDecoratedFeature(Feature<OreFeatureConfig>,OreFeatureConfig,Placement<CountRangeConfig>,CountRangeConfig)
  location: class Biome
2 errors
FAILURE: Build failed with an exception.```

--- a/build.gradle
+++ b/build.gradle
@@ -27,9 +27,8 @@ minecraft {
     // stable_#            Stables are built at the discretion of the MCP team.
     // Use non-default mappings at your own risk. they may not always work.
     // Simply re-run your setup task after changing the mappings to update your workspace.
-    //mappings channel: 'snapshot', version: "20200520-1.15.1"
     // makeObfSourceJar = false // an Srg named sources jar is made by default. uncomment this to disable.
-    mappings = "snapshot_20200519-1.15.1"
+    mappings channel: 'snapshot', version: "${mappingBuild}-${mappingMCVersion}"
 
     // Default run configurations.
     // These can be tweaked, removed, or duplicated as needed.

--- a/gradle.properties
+++ b/gradle.properties
@@ -2,3 +2,6 @@
 # This is required to provide enough memory for the Minecraft decompilation process.
 org.gradle.jvmargs=-Xmx5G
 org.gradle.daemon=false
+
+mappingBuild=20200519
+mappingMCVersion=1.15.1

--- a/gradle/wrapper/gradle-wrapper.properties
+++ b/gradle/wrapper/gradle-wrapper.properties
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.9-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.3-bin.zip

--- a/src/main/java/io/morefuelsmod/MFMOreGen.java
+++ b/src/main/java/io/morefuelsmod/MFMOreGen.java
@@ -70,10 +70,14 @@ public class MFMOreGen {
 		if(configSwitch == true){
     		for(Biome biome : ForgeRegistries.BIOMES.getValues()) {
     		//Use ArrayUtils to see if our Category[] wantedBiomes contains the current biome in the loop, and if so, generate the ore.
-    			if(ArrayUtils.contains(wantedBiomes, biome.getCategory())) {
+    			if(ArrayUtils.contains(wantedBiomes, biome.getCategory()))
+    			{
     				//This is how we actually add the ore generation to the biome(s)
-    					biome.addFeature(GenerationStage.Decoration.UNDERGROUND_ORES, Biome.createDecoratedFeature(Feature.ORE, new OreFeatureConfig(stoneType, wantedBlock.getDefaultState(), veinSize), Placement.COUNT_RANGE, cfg));
-    					MoreFuelsMod.LOGGER.info("MFM: Added an ore: " + wantedBlock + " to biome: " + biome.getCategory());	
+    				biome.addFeature(GenerationStage.Decoration.UNDERGROUND_ORES, Feature.ORE.withConfiguration(
+							new OreFeatureConfig(stoneType, wantedBlock.getDefaultState(), veinSize))
+							.withPlacement(Placement.COUNT_RANGE.configure(cfg)));
+
+					MoreFuelsMod.LOGGER.info("MFM: Added an ore: " + wantedBlock + " to biome: " + biome.getCategory());
     			}
     		}
     	} 
@@ -91,9 +95,12 @@ public class MFMOreGen {
     if(configSwitch == true) {
     	for(Biome biome : ForgeRegistries.BIOMES.getValues()) {
     		//For any biome that ISN'T the end of the nether, we're going to execute our oregen code
-    		if(!(biome.getCategory() == Biome.Category.THEEND || biome.getCategory() == Biome.Category.NETHER)) {
-    			//This is how we actually add the ore generation to the biome(s)
-    			biome.addFeature(GenerationStage.Decoration.UNDERGROUND_ORES, Biome.createDecoratedFeature(Feature.ORE, new OreFeatureConfig(stoneType, wantedBlock.getDefaultState(), veinSize), Placement.COUNT_RANGE, cfg));		
+    		if(!(biome.getCategory() == Biome.Category.THEEND || biome.getCategory() == Biome.Category.NETHER))
+    		{
+				//This is how we actually add the ore generation to the biome(s)
+				biome.addFeature(GenerationStage.Decoration.UNDERGROUND_ORES, Feature.ORE.withConfiguration(
+						new OreFeatureConfig(stoneType, wantedBlock.getDefaultState(), veinSize))
+						.withPlacement(Placement.COUNT_RANGE.configure(cfg)));
 			}
 		}
 		MoreFuelsMod.LOGGER.info("MFM: Added an ore: " + wantedBlock + " to all Overworld biomes");

